src/pages/Jungmin/Main/Feed/Feeds.js

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import Post from '../Post/Post';
+import './Feeds.scss';
+
+class FeedName extends React.Component {
+  render() {
+    return (
+      <div className="feedName">
+        <div className="feedUserIdandImg">
+          <img alt={this.props.alt} className="profil" src={this.props.src} />
+          <p>{this.props.feeduserId}</p>
+        </div>
+        <div className="moreicon">
+          <i className="fas fa-ellipsis-h"></i>
+        </div>
+      </div>
+    );
+  }
+}
+
+class Feed extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      howManyHeart: 1,
```

Comment:
```suggestion
      heartCount: 1,
```
불리언 값에 `is-` 네이밍을 쓰는게 아니라면 대체로 의문문을 변수명에 쓰는게 어색해보이네요

Reply:
수정했습니다 !